virtio-devices/src/vhost_user/fs.rs

```diff
@@ -482,10 +501,24 @@ impl VirtioDevice for Fs {
             kill_evt,
             pause_evt,
             slave_req_handler,
+            sockfd: Some(self.vu.as_mut().unwrap().as_raw_fd()),
+            disconnected: Some(self.disconnected.clone()),
+            activate_evt: Some(
+                self.activate_evt
+                    .try_clone()
+                    .map_err(ActivateError::CloneEventfd)
+                    .unwrap(),
+            ),
         });
 
         let paused = self.common.paused.clone();
         let paused_sync = self.common.paused_sync.clone();
+        /*
```

Comment:
Please drop this comment block. Unused code should be deleted before submitting.

Reply:
I removed it.